app/controllers/api/v1/garden_healths_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::V1::GardenHealthsController < ApplicationController
+  def index
+    return nil if GardenHealth.all.empty?
+    render json: GardenHealthSerializer.new(GardenHealth.all)
+  end
+
+  def show
+    return nil if !GardenHealth.exists? params[:id]
+    render json: GardenHealthSerializer.new(GardenHealth.find(params[:id]))
+  end
+
+  def create
+    new_record = GardenHealth.new(garden_health_params)
+    render json: GardenHealthSerializer.new(new_record) if new_record.save
+  end
+
+  def update
+    return nil if !GardenHealth.exists? params[:id]
+    updated_record = GardenHealth.find(params[:id])
+    updated_record.attributes = garden_health_params
+    render json: GardenHealthSerializer.new(updated_record) if updated_record.save
+  end
```

Comment:
Working on this endpoint made me question whether our models need more validations. I had to add a numericality validation to the `reading` attribute otherwise I could have updated it to be a random string and AR converted that to a float. Should we be validating datatypes when users update models? @lkriffell @hashmaster3k @D-Lessenden

Reply:
This is a great catch and would be good for a refactor. Do you want to make an issue on the project board?

Reply:
As far as validations of numericality, I didn't know we could test that so thats great.

Reply:
This is great! I was trying to do the same thing but anything I had in mind would have been way overcomplicated